api/db_queries.py

```diff
@@ -37,3 +37,31 @@
             RETURN { [category["category"]]: category.categoryname }
 )
 """
+
+query_items_for_menu = """
+FOR category IN menu_categories
+    FILTER category.language == @language
+    FOR catfile in category.files
+        FOR file in files
+            FILTER file._key == catfile
+            SORT file.filenr
+            RETURN {displayName: file.displayName,
+                    textname: file.textname,
+                    filename: file.filename,
+                    category: file.category}
+"""
+
+query_items_for_filter_menu = """
+FOR category IN menu_categories
+    FILTER category.language == @language
+    SORT category.categorynr
+    FOR catfile in category.files
+        RETURN {filename: catfile}
+"""
+
+query_items_for_category_menu = """
+FOR category IN menu_categories
+    FILTER category.language == @language
+    SORT category.categorynr
+    RETURN {category: category.category}
```

Comment:
ditto, we can just return a list of categories.

Reply:
Ditto as above.